sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1580,22 +1580,37 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Computes the length of a given string value
-   *
+   * Computes the length of a given string value.
+   * 
```

Comment:
Tarek Auel:  Instead of calling the Apache library, can you implement LevenshteinDistance by yourself?